### PR TITLE
added exercise1 html,css files

### PR DESCRIPTION
**Pull request by sumit-dahiya19**

### CODE REVIEW

File: exercise-1/style.css [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  margin: 40px 0px 0px -4px;
+  background-color: #ec3d67;
+  width: 313px;
+  border-radius: 34px;
+  height: 46px;
+  border: 0px;
+  color: white;
+  cursor: pointer;
+}
+.img1 {
```
Comment (gagan93jtg) [inline exercise-1/style.css]: img1,2,3 are not good and expressive class names. I can't infer from CSS that img2 is the username icon. Please fix this asap. I'll approve this PR then.

**Review by gagan93jtg — CHANGES_REQUESTED** [review]

File: exercise-1/style.css [hunk 2]
```diff
@@ -0,0 +1,41 @@
+body {
+  background-color: #3da2e0;
+}
+div {
+  background-color: white;
+  height: 620px;
```
Comment (gagan93jtg) [inline exercise-1/style.css]: Pls remove height and do without it

File: exercise-1/style.css [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  width: 620px;
+  border-radius: 10px;
+  margin: 44px 0px 0px 360px;
+}
+.name {
+  padding: 40px 118px 10px 13px;
```
Comment (gagan93jtg) [inline exercise-1/style.css:12]: I checked the boxes. They look very different. Please don't give unnecessary padding/margins here and there. Try to understand the design a little more.

**Review by gagan93jtg — CHANGES_REQUESTED** [review]

File: exercise-1/style.css (hunk 2 above)
Comment (sumit-dahiya19) [reply exercise-1/style.css]: Should i use any alternative for height or completely remove height parameter and work with default height of div

Comment (gagan93jtg) [reply exercise-1/style.css]: div has no default height. It changes height acc to the elements added into it. Let's stick to it's natural height.

Comment (sumit-dahiya19) [reply exercise-1/style.css]: Ok sir got it i have removed height parameter from div element and pushed the code again

**Review by gagan93jtg — APPROVED** [review]

**Pull request merged**